Detections/SecurityEvent/base64_encoded_pefile.yaml

```diff
@@ -24,15 +27,30 @@ relevantTechniques:
 query: |
 
   let ProcessCreationEvents=() {
-  let processEvents=SecurityEvent
+  let processEvents=(union isfuzzy=true
```

Comment:
The use of "processEvents" makes it hard to use where on EventData. When translating to ASIM we actually removed the let part.